packages/web-components/tests/e2e-storybook/cypress/integration/storybook-docs/storybook-docs.e2e.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable cypress/no-unnecessary-waiting */
+describe('storybook docs | default', () => {
+  it('should load all Storybook Docs without exceptions', () => {
+    cy.viewport(1280, 720);
+    cy.visit('/');
+    /**
+     * grab all components and get their href values which contain
+     * url to the corresponding docs
+     */
+    const components = cy.get('a[id*="explorercomponents"]');
+
+    components.each($component => {
+      const docsPath = $component.prop('href');
+
+      if (docsPath) {
+        const url = docsPath.split('?')[1].replace('path=/story', 'path=/docs');
+
+        cy.visit(`?${url}`);
```

Comment:
@annawen1 after messing around with Cypress a bunch, running a `cy.visit` multiple times in a single `it` test can cause some pretty bad memory issues. You should do an iteration and create multiple `it` tests instead.

Reply:
Case in point, Github Actions is pretty unhappy with this test:

![image](https://user-images.githubusercontent.com/1641214/151626802-f9728d02-c29c-4f1d-ba39-d69416e88a72.png)

Reply:
@jeffchew I ended up creating two tests, one to scrape all components from storybook and writing to the `fixtures/components.json` and then another one to import from that fixtures file to generate the tests. Looks like it's all passing! 🎆